style.css

```diff
@@ -130,3 +130,267 @@ p {
+}
+
+#submit {
+  margin-bottom: 2rem;
+}
+
+button:hover,
+#submit:hover {
+  background-color: #fff5e1;
+  color: #ff6b00;
+  cursor: pointer;
+}
+
+button:focus,
+#submit:focus {
+  background-color: #f4f1f0;
+  color: #5b5957;
+}
+
+button#resume {
+  margin: 1rem 0 15rem 2rem;
+  border: none;
+  font-family: "inter", sans-serif;
+  text-align: center;
+  color: #fff;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  display: inline-block;
+}
+
+#container-grid {
+  display: flex;
+  flex-direction: column;
+  background-color: #e5e5e5;
+}
+
+.card-work {
+  border: 5px;
+  text-align: center;
+}
+
+.button2 {
+  background-color: #fc6703;
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+  color: #fff;
+  border: none;
+  font-size: 1.1rem;
+  padding: 0.7rem;
+  width: 100%;
+}
+
+.language-button1 {
+  font-family: "Inter", sans-serif;
+  color: #091e42;
+  display: flex;
+  font-size: 1rem;
+  text-align: center;
+}
+
+.language-button1 li {
+  list-style-type: none;
+  border: 1px #8993a4 solid;
+  margin: 0 2px 5px 35px;
+  color: #fff;
+  padding: 3px 7px;
+}
+
+@media (max-width: 600px) {
+  .card-work {
+    background: url("./images/Img\ Placeholder.png");
+    background-repeat: no-repeat;
+    max-width: 750px;
+    margin: 0 auto;
+    width: 327px;
+    height: 450px;
+  }
+
+  .papd {
+    color: #fff;
+    font-family: "Crete Round", sans-serif;
+    font-size: 32px;
+    line-height: 44px;
+    padding: 0 3px 2px 40px;
+    width: 295px;
+    text-align: left;
+  }
+
+  .papd-p {
+    font-family: "inter", sans-serif;
+    font-size: 16px;
+    line-height: 24px;
+    color: #fff;
+    padding: 0 3px 0 40px;
+    width: 295px;
+    text-align: left;
+  }
+}
+
+/* About-me section */
+
+#about-me {
+  background-color: #1c1a19;
+  padding: 150px 2px 0 15px;
+}
+
+div#about-me-description {
+  background-image: url("images/illustration-about-me-mobile.png");
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  border-bottom: 1px solid #fff;
+}
+
+#skills {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+#languages-list,
+#skills-list,
+#frameworks-list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 2rem;
+  color: white;
+  font-family: "Inter", sans-serif;
+  font-size: 1.3rem;
+}
+
+#skills h3 {
+  color: white;
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+  margin: 3rem 0;
+}
+
+div.skills-title {
+  display: flex;
+  align-items: baseline;
+}
+
+.orange-icon {
+  width: 1.3rem;
+  height: 1.2rem;
+}
+
+#about-me-p {
+  font-size: 16px;
+  font-family: "inter", sans-serif;
+  line-height: 24px;
+  color: #f2f5f7;
+  width: 390px;
+}
+
+.title2 {
+  text-align: center;
+  font-size: 72px;
+  font-family: "Crete Round", sans-serif;
+  color: #ff6b00;
+  line-height: 96%;
+  letter-spacing: 0.37px;
+  vertical-align: top;
+  width: 387px;
+  padding-bottom: 50px;
+}
```

Comment:
- [ ] Please work on your entire style following the [figmal design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)
### ### **Take a look at your design and mobile view**
![mobile](https://user-images.githubusercontent.com/55596970/160210887-6809dd9d-d710-4382-97b1-ee754e9727b1.png)

Reply:
Please can you explain more about this

Reply:
Hi @Stanleeeeee 
Please watch the short [video](https://loom.com/share/00c6d4da96664c97810c33316f292094) and see how the project is displayed on mobile